cmd/solvercli/cmd/submit_order.go

```diff
@@ -0,0 +1,278 @@
+package cmd
+
+import (
+	"fmt"
+	"math/big"
+	"os/signal"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/skip-mev/go-fast-solver/shared/config"
+	"github.com/skip-mev/go-fast-solver/shared/contracts/fast_transfer_gateway"
+	"github.com/skip-mev/go-fast-solver/shared/contracts/usdc"
+	"github.com/skip-mev/go-fast-solver/shared/lmt"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	"golang.org/x/net/context"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+var submitCmd = &cobra.Command{
+	Use:   "submit",
+	Short: "Submit a new fast transfer order",
+	Long: `Submit a new fast transfer order through the FastTransferGateway contract.
+Example:
+  ./build/solvercli submit \
+  --config ./config/local/config.yml \
+  --token 0xaf88d065e77c8cC2239327C5EDb3A432268e5831 \
+  --recipient osmo13c9seh3vgvtfvdufz4eh2zhp0cepq4wj0egc02 \
+  --amount 1000000 \
+  --source-chain-id 42161 \
+  --destination-chain-id osmosis-1 \
+  --gateway 0x9c75534d7d6670a3ddd69a55b4067460f3e8744b \
+  --private-key 0xf6079d30f832f998c86e5841385a4be06b6ca2b0875b90dcab8e167eba4dcab1 \
```

Comment:
lmk if you have any concerns with this but since this is just for testing, i didnt think its worth including a more elaborate way to include the sender private key here (the keystore only has the keys of the solver)